activiti-api-process-model/src/main/java/org/activiti/api/process/model/events/BPMNMessageEvent.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2018 Alfresco, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.activiti.api.process.model.events;
+
+import org.activiti.api.model.shared.event.RuntimeEvent;
+import org.activiti.api.process.model.BPMNMessage;
+
+public interface BPMNMessageEvent extends RuntimeEvent<BPMNMessage, BPMNMessageEvent.MessageEvents> {
+
+    enum MessageEvents {
+        ACTIVITY_MESSAGE_WAITING,
+        ACTIVITY_MESSAGE_RECEIVED,
+        ACTIVITY_MESSAGE_SENT
```

Comment:
@CTI777 @igdianov, what if we replace `Activity` prefix by `BPMN` prefix? `BPMN_MESSAGE_WAITING`, `BPMN_MESSAGE_RECEIVED`, `BPMN_MESSAGE_SENT`. Conceptually `activity` is used to reference flow nodes that are not `events` or `gateways`.

Reply:
@erdemedeiros Goog catch! The naming convention we are using everywhere does not use ACTIVITI, i.e. 

```java
    enum TimerEvents {
        TIMER_SCHEDULED,
        TIMER_FIRED,
        TIMER_CANCELLED,
        TIMER_EXECUTED,
        TIMER_FAILED,
        TIMER_RETRIES_DECREMENTED

    }
```

Let's remove the ACTIVITI prefix to follow the same naming:

```java
enum MessageEvents {
        MESSAGE_WAITING,
        MESSAGE_RECEIVED,
        MESSAGE_SENT
}
```